app/models/slack_bot.rb

```diff
@@ -0,0 +1,30 @@
+class SlackBot
+  class << self
+    def post_to_recruitment(user, testing = false)
+      client.chat_postMessage(
+        { 
+          channel: testing ? "#progbot" : '#recruitment', 
+          attachments: [
+            {
+              pretext: 'New User Signup',
+              title: "A New User Has Signed Up: #{user.name}:::#{user.email}",
+              title_link: Rails.application.routes.url_helpers.admin_user_url(user, :only_path => false),
+              text: '',
+              color: '#7CD197'
+            }
+          ]
+        }.merge(default_params)
```

Comment:
This would never cause a problem here, but interesting to note (in case you someday find yourself trying to eek out the tiniest of performance improvements or to squeeze a memory footprint down a smidge):

`#merge` will leave you with three hashes
```
hash1 = {}
hash2 = {}
hash1.merge(hash2) #=> returns a third hash
```

```
hash1 = {}
hash2 = {}
hash1.merge!(hash2) #=> still just two hashes
```